test/e2e/framework/test_context.go

```diff
@@ -52,10 +52,10 @@ func RegisterCommonFlags() {
 	config.GinkgoConfig.RandomizeAllSpecs = true
 
 	flag.StringVar(&TestContext.KubeHost, "kubernetes-host", "http://127.0.0.1:8080", "The kubernetes host, or apiserver, to connect to")
-	flag.StringVar(&TestContext.KubeConfig, "kubernetes-config", os.Getenv(clientcmd.RecommendedConfigPathEnvVar), "Path to config containing embedded authinfo for kubernetes.")
+	flag.StringVar(&TestContext.KubeConfig, "kubernetes-config", os.Getenv(clientcmd.RecommendedConfigPathEnvVar), "Path to config containing embedded authinfo for kubernetes. Default value is from environmen variable "+clientcmd.RecommendedConfigPathEnvVar)
```

Comment:
missing 't' on 'environmen'.  And can you quote the env var name so it stands out as a special word?